Makefile

```diff
@@ -119,7 +119,7 @@ archive:
 release: package archive portable_zip
 
 docker_test:
-	docker run -v `pwd`:`pwd` -w `pwd` --name swiftlint --rm norionomura/swift:42 swift test --parallel
+	docker run -v `pwd`:`pwd` -w `pwd` --name swiftlint --rm norionomura/swift:421 swift test --parallel
```

Comment:
there's an issue on Linux when compiling with 4.2 which is fixed on 4.2.1 (should we highlight this bump somewhere)?